src/components/content/Content.js

```diff
@@ -29,6 +29,11 @@ export default class ContentComponent extends Component {
     return ContentComponent.schema();
   }
 
+  constructor(component, options, data) {
+    super(component, options, data);
+    this.options.noDefaults = true;
```

Comment:
```this.options``` is a globally shared options object. What this means is that when you set it here, it will set it for ALL components. Is this intended?